modules/Core/src/main/java/org/terasology/core/world/generator/facetProviders/PerlinBaseSurfaceProvider.java

```diff
@@ -55,7 +55,7 @@ public void process(GeneratingRegion region) {
         float[] noise = surfaceNoise.noise(processRegion);
 
         for (int i = 0; i < noise.length; ++i) {
-            noise[i] = seaLevel + seaLevel * ((noise[i] + 1f) / 2f);
+            noise[i] = seaLevel + seaLevel * ((noise[i] * 2.11f + 1f) / 2f);
```

Comment:
Yes, exactly! The original implementation had a range of -2.11 to +2.11 for 8 octaves and -2.12 to +2.12 for 9 octaves. The influence of additional layers (octaves) becomes exponentially smaller.

I think that the scale could be smaller (and less octaves) to get a less extreme environment, but world shaping was not part of this.